pkgs/development/python-modules/torch/bin.nix

```diff
@@ -45,21 +67,21 @@ in buildPythonPackage {
     requests
     setuptools
     typing-extensions
+    sympy
+    jinja2
+    networkx
+    filelock
+  ] ++ lib.optionals stdenv.isx86_64 [
+    openai-triton
   ];
 
   postInstall = ''
     # ONNX conversion
     rm -rf $out/bin
   '';
 
-  postFixup = let
-    rpath = lib.makeLibraryPath [ stdenv.cc.cc.lib ];
-  in ''
-    find $out/${python.sitePackages}/torch/lib -type f \( -name '*.so' -or -name '*.so.*' \) | while read lib; do
-      echo "setting rpath for $lib..."
-      patchelf --set-rpath "${rpath}:$out/${python.sitePackages}/torch/lib" "$lib"
-      addOpenGLRunpath "$lib"
-    done
+  postFixup = ''
+    addAutoPatchelfSearchPath "$out/${python.sitePackages}/torch/lib"
```

Comment:
It seems that this takes priority over `cuda_cudart`, thus we're losing the `libnvrtc-builtins.so` fix.

I suggest we:

- `rm $out/${python.sitePackages}/lib/libnvrc*`
- cherry-pick https://github.com/SomeoneSerge/nixpkgs/commit/f55d2a112dc33855bad42980bf93d684505cbe6d so that we don't have to wait for [staging](https://nixpk.gs/pr-tracker.html?pr=226038)

Reply:
Additionally, we should verify that none of `$out/${python.sitePackages}/torch/*.so` try to `dlopen()` any of `$out/${python.sitePackages}/torch/lib/*.so` without declaring them in DT_NEEDED

Reply:
Tracking in https://github.com/NixOS/nixpkgs/issues/227748